tests/Brahma.FSharp.Tests/CompositeTypesTests.fs

```diff
@@ -0,0 +1,138 @@
+﻿module CompositeTypesTests
+
+open Expecto
+open Brahma.FSharp.OpenCL
+open FSharp.Quotations
+open Brahma.FSharp.Tests
+
+#nowarn "667"
```

Comment:
I guess it would be nice to add a comment with the reason why specific warnings are turned off.